### ISSUE TITLE
Mesh is stretched and deformed When Import Runtime

### ISSUE THREAD

**Issue by triansion** [atteneder/glTFast#419]
Hello:

When I load a local GLB through the interface GltfImport.LoadGltfBinary or GltfImport.Load at runtime, the loaded mesh is stretched and deformed.But when I manually drag the GLB into the editor's scene the mesh is normal.

**Screenshots**
This is RunTime:
![image](https://user-images.githubusercontent.com/48820794/174555912-7b7c4a3c-5b12-449a-9c29-148510647f93.png)

This is Manually:
![image](https://user-images.githubusercontent.com/48820794/174556170-c095b4f6-7f35-4294-aa02-c2a278e398dd.png)

**Desktop (please complete the following information):**
 - glTFast 
 - Unity Editor version :2020.3.0.f1c1
 - Render Pipeline and version : Universal Render Pipeline 10.3.2
 - Platform: Editor

**Comment by atteneder** [comment]
Hi @triansion ,

Thanks for reporting! Can your provide a test file? Have you [validated your file](https://github.khronos.org/glTF-Validator/)? What's the origin of the file/which software was used to generate it?

I suspect your file has non-ordered and/or non-normalized bone weights. You can check this by increasing the bone count in the quality settings to 4 (or more).

For editor import, re-sorting bone weights is always performed. To enable it at runtime, add  `GLTFAST_SAFE` to the scripting defines in the player settings (see [safe mode](https://github.com/atteneder/glTFast/blob/main/Documentation%7E/ProjectSetup.md#safe-mode)). Note that this has a minor performance cost though.

hth

**Comment by triansion** [comment]
Thanks very much! It works fine after i added GLTFAST_SAFE to the scripting defines.But when I play skeletal animation with blendshapes,it is still stretched and deformed. like this:
![image](https://user-images.githubusercontent.com/48820794/176334704-7120273b-dea1-444e-ba8d-2e02512d3e96.png)


GLB.json file is there:
[GLB.zip](https://github.com/atteneder/glTFast/files/9006655/GLB.zip)

Thanks again for your reply

**Comment by triansion** [comment]
this is GLB zip file :
[GLB.zip](https://github.com/atteneder/glTFast/files/9006665/GLB.zip)

**Comment by triansion** [comment]
And this is animation related files :🤣
[Animation.zip](https://github.com/atteneder/glTFast/files/9006683/Animation.zip)

**Comment by atteneder** [comment]
@triansion Thanks, I can reproduce the issue.

Any chance you have a version that's not meshopt compressed (no gltfpack ran on it)? It's surprisingly painful to inspect data otherwise :)

**Comment by atteneder** [comment]
@triansion Thanks!

So the uncompressed one works as intended here. I tried to compress it myself like so:

```bash
gltfpack -c -i uncompressed.glb -o meshopt.glb
gltfpack -cc -i uncompressed.glb -o meshopt2.glb
gltfpack -noq -c -i uncompressed.glb -o meshopt-noq.glb
gltfpack -noq -cc -i uncompressed.glb -o meshopt-noq2.glb
``` 

All of those worked. AFAIK recently gltfpack started sorting joints by weight, so maybe that's it. Which gltfpack version did you use? I used 0.17 (maybe it's actually custom build that's slightly newer; have to check)

If you can confirm this, then there seems to be a bug in the joint by-weight sorting.

**Comment by atteneder** [comment]
@triansion You're right about the parenthesis bug. After fixing it the initial, compressed model seems to work just fine again. Can you confirm this?

Awesome, you found it :)

**Comment by triansion** [comment]
@atteneder Yes,it works fine after I fixed  the parenthesis bug! Thanks for your help😁😁

**Comment by atteneder** [comment]
@triansion Excellent! The fix is in already in 4.8.3. Thanks again!
